app/config/config_prod.yml

```diff
@@ -2,10 +2,10 @@ imports:
     - { resource: config.yml }
 
 #framework:
-#    validation:
-#        cache: validator.mapping.cache.doctrine.apc
-#    serializer:
-#        cache: serializer.mapping.cache.doctrine.apc
+#    cache:
+#        pools:
```

Comment:
here it's `adapter` and requires https://github.com/symfony/symfony/pull/18625 beforehand.
Otherwise, :+1: